integration_tests/test_cargo_audit.py

```diff
@@ -0,0 +1,10 @@
+import os
+
+# cargo-audit reads and checks for the presence of Cargo.lock
```

Comment:
How about instead of checking for the existence of the `Cargo.lock` file, we just do a `cargo build --all-features --release` before and then call `cargo audit`?

One of the tests of the rust-vmm-ci is checking the build anyway, so we will end up always in 2 situations:
- the build command was already run and `cargo build` will be a no-op.
- the build command was not run, and the build will happen as part of this command; when the build command will be run next time, it will be a no-op.

Running `cargo build` and then `cargo audit` should not incur a significant delay.